src/maps/workPoolQueue.ts

```diff
@@ -31,6 +32,7 @@ export const mapWorkPoolQueueToWorkPoolQueueResponse: MapFunction<WorkPoolQueue,
     is_paused: source.isPaused,
     concurrency_limit: source.concurrencyLimit,
     priority: source.priority,
+    last_polled: this.map('Date', source.lastPolled, 'string'),
     status: source.status.toUpperCase() as Uppercase<WorkPoolQueueStatus>,
   }
 }
```

Comment:
This mapper doesn't make sense to me, when are we mapping a client-side work queue back to an api-side object 🤔

Reply:
y'know. i thought that too. 

and a quick search _looks_ like i can just cull this. will double-check slash remove. brb

Reply:
killed that. there was on for `WorkQueue` -> `WorkQueueResponse` also 🤨 . killed that one too.